projects/observability/src/shared/graphql/request/handlers/spans/span-graphql-query-handler.service.ts

```diff
@@ -58,7 +58,7 @@ export class SpanGraphQlQueryHandlerService implements GraphQlQueryHandler<Graph
   protected buildTimeRange(timestamp?: Date): GraphQlTimeRange {
     let timeRange;
     if (timestamp) {
-      const duration = new TimeDuration(30, TimeUnit.Second);
+      const duration = new TimeDuration(60, TimeUnit.Second);
       timeRange = {
         startTime: timestamp.getTime() - duration.toMillis(),
         endTime: timestamp.getTime() + duration.toMillis()
```

Comment:
Do we also need to set seconds & millis to zero here like we did in #104?

Reply:
No we don't, as we are searching in a range of time here, so this won't be an issue.